File: POO/CS/OzCorp/OzCorp/Business/Secretary.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OzCorp.Business
{
    public class Secretary : Employee
    {
        public Secretary(string Name, string RG, long CPF, string Enrollment, string Email, string Password, BloodType Blood, Gender Gender) : base(Name, RG, CPF, Enrollment, Email, Password, Role.GetRole(Roles.Secretary), Blood, Gender)
        {
        }
    }
}

File: POO/J/OzCorp/src/br/com/ozcorp/business/Secretary.java
package br.com.ozcorp.business;
public class Secretary extends Employee {
	public Secretary(String name, String rg, long cpf, String enrollment, String email, String password, 
			BloodType blood, Gender gender) {
		super(name, rg, cpf, enrollment, email, password, Role.Secretary, blood, gender);
	}
}
